Repositories/IRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace AirportTicketBooking.Repositories
+{
+    public interface IRepository<T>
+    {
+        void Save(T data);
+
+        List<object> Load();
```

Comment:
let's specify a generic type here so we can use define a special type in each repo directly.

Reply:
sorry, but I didn't understand what I must do. I already use a generic type